### ISSUE TITLE
Accessibility issues in category tabs

### ISSUE THREAD

**Issue by Waakul** [ScratchAddons/ScratchAddons#7278]
### Why this would be helpful

![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/5e040c82-c86f-4078-99c5-11dd2be70aa6)
As you see here, the background of the tab is hard to distinguish from the background and the search toolbar background.

### My suggestion

Solution:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/a977a3c0-b394-433b-851b-2f869ee72e63)
use something like this, margin to distinguish and border radius so it's easy to follow the border and also for consistent design lang

### Possible alternatives

i dont have any other ideas

### Additional context

![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/08e338c9-1578-4baa-8fe5-bbb5efc98759)

**Comment by Waakul** [comment]
![image](https://github.com/ScratchAddons/ScratchAddons/assets/135821321/5de23df0-7de5-4aaf-a1fa-5fdcd1a759cb)
The case is even worse in the org design :/

**Comment by WorldLanguages** [comment]
Isn't the indicator on the side good enough to know which tab is selected?

**Comment by Waakul** [comment]
> Isn't the indicator on the side good enough to know which tab is selected?

I don't think so... It is a bit unnoticeable. even the indicator on the side is too close to the background.

**Comment by WorldLanguages** [comment]
Or is this about the contrast between the 3 different shade of gray backgrounds that you can see in the whole settings page? I got confused because the screenshots barely show those parts of the page.

**Comment by DNin01** [comment]
I do think shifting the orange bar on the side that indicates the selected tab further to the right like you did does help it stand out more. It was always kind of hard to see for me at the very edge of the screen.

I guess the highlight (darker background) might stand out slightly more because it now has that margin that sort of creates a border that makes it pop out more?

**Comment by DNin01** [comment]
> As you see here, the background of the tab is hard to distinguish from the background and the search toolbar background.

I guess you could say that, but there's also the shade of the other tabs above and below to contrast. Also, tabs sometimes take on the background color of the content, to make it look like the tab is attached to it.

**Comment by mxmou** [comment]
This isn't an accessibility issue - the orange bar already has 3:1 contrast with the background (except in light mode, but the linked PR doesn't solve that). The background of the selected item doesn't have much contrast (even with the darker color proposed in the PR), and it doesn't need to (because the selection is also indicated by the orange bar).

**Comment by DNin01** [comment]
Also, if you have trouble with edge pixels or have a display with a [safe area](https://en.wikipedia.org/wiki/Safe_area_(television)), you can always shrink your window.
